src/openforms/template/backends/sandboxed_django.py

```diff
@@ -47,3 +50,27 @@ def get_templatetag_libraries(self, custom_libraries: dict) -> dict:
 """
 An instance of the 'sandboxed' Django templates backend.
 """
+
+
+def get_openforms_backend():
+    return SandboxedDjangoTemplates(
+        {
+            "OPTIONS": {
+                "builtins": [
+                    "openforms.emails.templatetags.appointments",
+                    "openforms.emails.templatetags.form_summary",
+                    "openforms.emails.templatetags.payment",
+                    "openforms.emails.templatetags.products",
+                    "openforms.config.templatetags.privacy_policy",
+                ],
+            }
+        }
+    )
+
+
+openforms_backend = cast(
+    SandboxedDjangoTemplates, SimpleLazyObject(get_openforms_backend)
```

Comment:
need to make it lazy because otherwise we get a bunch of circular imports (models.py files import the backend). Apparently instantiating the backend completely tries to load all the builtins.

I think this builtins situation will see some refactoring in the future anyway.

Reply:
Casting this *is* a lie though... It serves for IDE auto-complete purposes. But it breaks type safety. It **should** be a `LazyObject[SandboxedDjangoTemplates]`, but the django annotations are outside of this projects scope.